packages/scalars/vite.config.ts

```diff
@@ -0,0 +1,46 @@
+import { defineConfig } from 'vite';
+import dts from 'vite-plugin-dts';
+import generateFile from 'vite-plugin-generate-file';
+
+const entry = {
+    index: 'src/index.ts',
```

Comment:
Would be nice to dynamically generate an entry for each scalar and also add it as an export in the generated package.json files.
Then it would be possible to do `import { schema, config } from "@powerhousedao/scalars/email-address".